examples/README.md

```diff
@@ -189,7 +249,11 @@ example.
 [compiler-cranelift]: ./compiler_cranelift.rs
 [compiler-llvm]: ./compiler_llvm.rs
 [cross-compilation]: ./engine_cross_compilation.rs
+[exported-global]: exports_global.rs
```

Comment:
The other files start with `./`, I have no idea if that's significant for GitHub markdown links, but might be good to make this consistent